INSTALL.md

```diff
@@ -0,0 +1,126 @@
+
+### CLMM and dependency installation
+
+Now, you can install CLMM and its dependencies as
+
+```bash
+    pip install numpy scipy astropy matplotlib
+    pip install pytest sphinx sphinx_rtd_theme
+    pip install jupyter  # need to have jupyter notebook tied to this environment, you can then see the environment in jupyter.nersc.gov
+    git clone https://github.com/LSSTDESC/CLMM.git  # If you'd like to contribute but don't have edit permissions to the CLMM repo, see below how to fork the repo instead.
+    cd CLMM   
+    python setup.py install     # build from source
+```
+
+## Access to the proper environment on cori.nersc.gov <a name="access_to_the_proper_environment_on_cori"></a>
+
+If you have access to NERSC, this will likely be the easiest to make sure you have the appropriate environment.  After logging into cori.nersc.gov, you will need to execute the following.  We recommend executing line-by-line to avoid errors:
+
+```bash
+    module load python  # Also loads anaconda
+    conda create --name clmmenv  # Create an anaconda environment for clmm
+    source activate clmmenv  # switch to your newly created environment
+    conda install pip  # need pip to install everything else necessary for clmm 
+    conda install ipython # need to have the ipython tied to this environment
+    conda install -c conda-forge firefox  # Need a browser to view jupyter notebooks  
+```
+
+Note, for regular contributions and use, we recommend adding `module load python` to your `~/.bashrc` so you have anaconda installed every time you log in.  You will subseqeuntly also want to be in the correct environment whenever working with `clmm`, which means running `source activate clmmenv` at the start of each session.
+
+Once in your CLMM conda env, you may follow the [basic procedure](#basic_install) to install CLMM and its dependencies.
+
+The above allows you to develop at NERSC and run pytest.  Your workflow as a developer would be to make your changes, do a `python setup.py install` then `pytest` to make sure your changes did not break any tests.
+
+If you are a DESC member you may also add to your CLMM environment the GCR and GCRCatalog packages to access the DC2 datasets at NERSC. To run the DC2 example notebooks provided in CLMM, the following need to be installed in your CLMM environment at NERSC. Once in your CLMM environment (`source activate clmmenv`), run
+
+```bash
+    pip install pandas
+    pip install pyarrow
+    pip install healpy
+    pip install h5py
+    pip install GCR
+    pip install https://github.com/LSSTDESC/gcr-catalogs/archive/master.zip
+    pip install https://github.com/yymao/FoFCatalogMatching/archive/master.zip
```

Comment:
or simply `pip install FoFCatalogMatching`. I tried on nersc and it is working fine.

Reply:
Changed.